inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -141,15 +145,32 @@ def simulate_async(self,
         self.output_time_step = output_time_step
         self.n_cores = n_cores
 
+        commands = self.get_commands()
+
         task_id = super().simulate_async(
             simulator,
             resource_pool_id=resource_pool_id,
             n_cores=n_cores,
-            method_name="simpleFoam",
+            commands=commands,
         )
 
         return task_id
 
+    def get_commands(self):
+        """Returns the commands for the simulation.
+        """
+        templates_path = os.path.join(SCENARIO_TEMPLATE_DIR,
+                                      OPENFOAM_TEMPLATE_INPUT_DIR)
+        commands_file_path = os.path.join(templates_path, "commands.json")
```

Comment:
Should this be a global variable as well? Or just remove jinja extension from this `COMMANDS_TEMPLATE_NAME` env variable since I assume they should be similar.